guides/common/modules/proc_adding-a-default-http-proxy-by-using-cli.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-a-default-http-proxy-by-using-cli"]
+= Adding a default HTTP proxy by using CLI
+
+You can use Hammer CLI to add a default HTTP proxy to {Project}.
+
+.Procedure
+. Verify that the `http_proxy`, `https_proxy`, and `no_proxy` variables are not set:
++
+[options="nowrap"]
+----
+# unset http_proxy https_proxy no_proxy
+----
```

Comment:
Prerequisite?
For your consideration.

`* Ensure that...`

Reply:
Moved.